### PR TITLE
Add a form to move notes to other folders.

### PR DESCRIPTION
**Pull request by edditler**
Could also be used to address this issue https://github.com/archivy/archivy/issues/223.

Bildschirmfoto 2021-05-08 um 10 36 54

### CODE REVIEW

**Comment by edditler** [comment]
There is a `Path` attribute in the front matter. Is it possible to reuse the `api.update_dataobj_frontmatter` to change that too? I don't think the attribute is used anywhere in the code.

**Comment by Uzay-G** [comment]
I think it's best if we deprecate that attribute, it's a bit redundant.

File: archivy/data.py [hunk 1]
```diff
@@ -134,6 +134,15 @@ def get_item(dataobj_id):
     return None
 
 
+def move_item(dataobj_id, new_path):
+    """Move dataobj of given id to new_path"""
+    file = get_by_id(dataobj_id)
+    if move(file, f"{get_data_dir()}/{new_path}/"):
```
Comment (Uzay-G) [inline archivy/data.py:140]: we can just change this part to `return move(file, f"{get_data_dir()}/{new_path}/")`

Comment (edditler) [reply archivy/data.py:140]: right.

**Comment by edditler** [comment]
> I think it's best if we deprecate that attribute, it's a bit redundant.

Should we do that in this PR or another one?

**Comment by edditler** [comment]
I was a bit surprised about the `from shutils import ...`. Maybe it's clearer to just `import shutils`?

**Comment by Uzay-G** [comment]
Yeah good idea.

**Comment by Uzay-G** [comment]
> > I think it's best if we deprecate that attribute, it's a bit redundant.
> 
> Should we do that in this PR or another one?
Yeah don't worry about it.

**Comment by Uzay-G** [comment]
@edditler I made some changes to the design (see below) and refactored some stuff. Does this design work for you (a bit simpler):
image

**Comment by Uzay-G** [comment]
Alright I also added some tests! Once I have your go I'll merge this in :)

**Review by edditler — COMMENTED** [review]
Works for me, thanks!

**Comment by Uzay-G** [comment]
Perfect :)

I'll make a new release soon, once the other open PRs are sorted.

**Pull request merged**